lasagna-master/hemartin/lasagna_master.go

```diff
@@ -0,0 +1,70 @@
+package lasagna
+
+const noodleQuantityPerLayer int = 50
+const sauceQuantityPerLayer float64 = 0.2
+const originalQuantityPortions float64 = 2.0
```

Comment:
Good idea to make constants for these values. I copied this to my solution

Reply:
Thanks Sam!